tutorials/applications/QAOA and QAA to solve a MIS problem.ipynb

```diff
@@ -4,12 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    np.random.seed(123)
</pre><p>Maybe add a short comment saying this is to ensure we have repeatable results on the tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>